lib/datadog/remote-config/examples/fetch.rs

```diff
@@ -0,0 +1,36 @@
+use anyhow::{Context, Result};
+use remote_config::{Client, Config};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let api_key = std::env::var("DD_API_KEY").context("missing DD_API_KEY env var")?;
+    let app_key = std::env::var("DD_APP_KEY").context("missing DD_APP_KEY env var")?;
```

Comment:
Does this separately need an application key? IIRC that requirement has been deprecated recently (generally).